src/quo2/components/list_items/channel.cljs

```diff
@@ -0,0 +1,47 @@
+(ns quo2.components.list-items.channel
+  (:require [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.counter.counter :as quo2.counter]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.avatars.channel-avatar :as channel-avatar]
+            [quo2.components.markdown.text :as quo2.text]))
+
+(defn list-item [{:keys [name lock-status mentions-count unread-messages? muted is-active-channel? emoji channel-color] :or {channel-color colors/primary-50}}]
+
+  [rn/view {:style (merge {:height          48
+                           :display         :flex
+                           :border-radius   12
+                           :flex-direction  :row
+                           :justify-content :space-between
+                           :align-items     :center
+                           :width           "100%"
+                           :padding-left    12
+                           :padding-right   12}
+                          (when is-active-channel? {:background-color (colors/theme-alpha channel-color 0.05 0.05)}))}
+
+   [rn/view {:display :flex
+             :flex-direction :row
+             :justify-content :flex-start
+             :align-items :center}
+    [channel-avatar/channel-avatar
+     {:big?                   true
+      :lock-status            lock-status
+      :emoji-background-color (colors/theme-alpha channel-color 0.1 0.1)
+      :emoji                  emoji}]
+    [quo2.text/text {:style (merge {:margin-left 12} (when  (and (not (= lock-status :locked))  muted) {:color colors/neutral-40}))
+                     :weight :medium :size :paragraph-1} (str "# " name)]]
+
+   [rn/view {:style {:height 20}}
+
+    (when (and (not (= lock-status :locked))  muted) [quo2.icons/icon :main-icons2/muted {:size            20
+                                                                                          :no-color         true}])
+    (when (and (not (= lock-status :locked)) (not muted)  (pos? (int mentions-count)))
+      [rn/view {:style {:margin-right 2
+                        :margin-top 2}}
+
+       [quo2.counter/counter {:override-bg-color channel-color} mentions-count]])
+
+    (when (and (not (= lock-status :locked)) (not muted) (not (pos? (int mentions-count))) unread-messages?)
```

Comment:
[Readability] I find this line almost impossible to parse in my head. If you break the lines the conditions will become much more obvious.

```clojure
(and (not (= lock-status :locked))
     (not muted)
     (not (pos? (int mentions-count)))
     unread-messages?)
```